BrightID/src/components/Connections/ConnectionScreenController.tsx

```diff
@@ -65,14 +65,29 @@ function ConnectionScreenController() {
     }, [api, connectionId]),
   );
 
-  // Update connection verifications in store
+  // Update connection verifications
   useEffect(() => {
     if (connectionProfile) {
       console.log(`Updating verifications for ${connectionProfile.id}`);
+      const texts = [];
```

Comment:
The code to get the verification texts from a verifications array should be a function somewhere in utils. So the same code can be used here and in HomeScreen.